src/blueapi/cli/cli.py

```diff
@@ -182,6 +183,22 @@ def store_finished_event(event: WorkerEvent) -> None:
             finished_event.append(event)
 
     parameters = parameters or "{}"
+    schema: PlanModel = client.get_plan(name)
+    progress_tracking = f"Trying to run plan: {name} with the expected schema: {schema}"
+    print(progress_tracking)
```

Comment:
I'm not very familiar with using the CLI and how noisy/verbose it is but this function is now printing quite a lot of output which is arguably debug/verbose info.

Reply:
got a variation of the latter

Reply:
I already got this highlight of what does not match

Reply:
![image](https://github.com/DiamondLightSource/blueapi/assets/56644812/6f37393d-76e4-4c50-8f53-4d2fa6d85d93)
the latest branch looks diff

Reply:
I don't think a new ticket is needed at this point